script/vsts/nightly-release.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         displayName: Download Release Artifacts
 
       - script: |
-          node $(Build.SourcesDirectory)\script\vsts\upload-artifacts.js --create-github-release --assets-path "$(System.ArtifactsDirectory)" --linux-repo-name "atom"
+          node $(Build.SourcesDirectory)\script\vsts\upload-artifacts.js --create-github-release --assets-path "$(System.ArtifactsDirectory)" --upload-to-s3 "skip"
```

Comment:
Setting an Azure variable is better than a flag because our code with upstream will be the same. Now we should maintain our version of the code here.

Reply:
Something like: `SHOULD_Upload_TO_S3`